Expensify/App
# Refactor BankAccount step copy and actions

### Details


### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/179278
$ https://github.com/Expensify/Expensify/issues/179277

### Tests
1. Login to an account with a Workspace, but without a verified bank account.
2. Navigate to `Settings > Select Workspace > Expensify Card > Get started`
3. Verify that the modal you see looks like the screenshots below.
4. Verify that clicking the links open the correct pages and clicking the buttons open the plaid modal or move you to the manual connection step.

### QA Steps
Steps above.

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
